config/desktop-development.json

```diff
@@ -105,7 +105,6 @@
 		"perfmon": false,
 		"persist-redux": true,
 		"plans/personal-plan": true,
-		"post-editor-github-link": false,
```

Comment:
For completeness: this has never been in use since the InitialCommit (53bde8c3f9)